src/cliActions.js

```diff
@@ -136,9 +136,17 @@ const streamDApp = ({ pw }, signingKeyFile, devMode) =>
         // read signing key
         const rawSigningKey = fs.readFileSync(signingKeyFile, 'utf8');
         const signingKey = JSON.parse(rawSigningKey);
-        watchAndStreamBundleData(devMode, signingKey, ({ content }) => {
-          console.log({ content, singingPrivateKey, signingKey }); // TODO: need to process result
-        });
+        watchAndStreamBundleData(
+          devMode,
+          { ...signingKey, singingPrivateKey },
+          ({ dAppBundle }) => {
+            const secretKey = Buffer.from(singingPrivateKey.toString(), 'hex');
+            const signedDAppBundle = Buffer.from(
+              tweetnacl.sign(Buffer.from(dAppBundle, 'utf8'), secretKey),
```

Comment:
why are you signing the DApp bundle here?

Reply:
I mean you are signing an json object of am I wrong?

Reply:
yes i sign the json object

Reply:
ah okay, i already sign the hash, but i think you want to sign the dapp bundle too, so i will remove this part, the hash is already sign